src/PublicApiGenerator/NullableContext.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Mono.Cecil;
+using System;
+using System.Collections.Generic;
+
+namespace PublicApiGenerator
+{
+    // https://github.com/dotnet/roslyn/blob/master/docs/features/nullable-metadata.md#nullablecontextattribute
+    internal class NullableContext
+    {
+        [ThreadStatic]
+        private static Stack<ICustomAttributeProvider> _nullableContextProviders;
```

Comment:
Isn't the thread static in our case overkill?

Reply:
Nothing works without it. After all, nothing prevents calling api generation in parallel, for example, as the default xUnit do. If you remove this code, then with a probability of 99.9%, almost all tests fail. An alternative is to pass providers as arguments to the methods, but this is inconvenient and complicates the code.

Reply:
Ah yeah forgot the parallel execution.